Update workspace configuration doc (#5696)
skypilot-org/skypilot: docs/source/admin/workspaces.rst

         disabled: true

.. dropdown:: Schema of the ``workspaces`` section:

   .. code-block:: yaml

      workspaces:
         # Workspace 'default' is created by SkyPilot and is used if no workspaces
         # are defined. Admins can optionally override settings for this workspace.
         default: {}

         <workspace name>:
            <infra name>:  # aws, gcp, ...; ssh; k8s
               disabled: false  # Disable an infra provider for this workspace (default: false)

            # Currently, 'gcp' supports an additional field:
            gcp:
               disabled: false

>                project_id: GCP project ID

Reviewer: This should have the same indentation?

To apply the configuration, follow the following steps:

.. tab-set::

   .. tab-item:: Helm Deployment

>       If your use :ref:`Helm Deployment <sky-api-server-helm-deploy-command>` for your API server, apply the workspace configuration to your local config file and run:

Reviewer: ```suggestion
      If you used :ref:`Helm Deployment <sky-api-server-helm-deploy-command>` for your API server, include workspace configuration to in local SkyPilot config file and run:
```

      .. code-block:: bash

         # RELEASE_NAME and NAMESPACE are the same as the ones used in the helm deployment
         helm upgrade --install $RELEASE_NAME skypilot/skypilot-nightly --devel \
            --namespace $NAMESPACE \
            --reuse-values \
            --set-file apiService.config=/your/path/to/config.yaml
      

>       To apply a new workspace configuration, update the config file and run the same command again. The API server will reload the new configuration automatically. For more details, refer to :ref:`Setting the SkyPilot config in Helm Deployment <sky-api-server-config>`

Reviewer: ```suggestion
      To change workspace configuration, update the config file and run the same command again. The API server will reload the new configuration automatically with no downtime. For more details, refer to :ref:`Setting the SkyPilot config in Helm Deployment <sky-api-server-config>`
```

   .. tab-item:: VM Deployment

>       If you are using a :ref:`VM Deployment <sky-api-server-cloud-deploy>` for your API server or testing workspaces locally, edit the workspace configuration in the :ref:`SkyPilot config file <config-yaml>` on the **API server host** directly. API server will automatically reload the configuration to apply the changes.

Reviewer: ```suggestion
      If you used a :ref:`VM Deployment <sky-api-server-cloud-deploy>` for your API server or testing workspaces locally, edit the workspace configuration in the :ref:`SkyPilot config file <config-yaml>`, `~/.sky/config.yaml`. API server will automatically reload the configuration to apply the changes.
```
